### PR TITLE
mssu theme

### PR DESCRIPTION
**Pull request by DonatoBD**
Adding theme folder for mssu site as requested in #15827

### CODE REVIEW

**Review by jignaciopm — COMMENTED** [review]
Are the .DS_Store files necessary?

File: edx-platform/mssu-produhktiv/lms/static/sass/partials/lms/theme/_extras.scss [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+.wrapper-course-material.courseware .course-material {
+  overflow: auto;
+}
+
+.course-content .xmodule_display.xmodule_SequenceModule .sequence-nav .sequence-list-wrapper {
+  overflow: inherit;
+}
+
+//profile
+  .view-profile .badge-set-display .badge-display {
```
Comment (jignaciopm) [inline edx-platform/mssu-produhktiv/lms/static/sass/partials/lms/theme/_extras.scss]: Identation!

Comment (DonatoBD) [reply edx-platform/mssu-produhktiv/lms/static/sass/partials/lms/theme/_extras.scss]: Ready :)

**Review by jignaciopm — COMMENTED** [review]
Did you override all .html?

**Comment by DonatoBD** [comment]
.DS_Store file deleted :)

I only changed one static (banner inside lms/static ), for now everything else remains the same as the msus theme

**Review by NeOneSoft — APPROVED** [review]

**Review by jignaciopm — APPROVED** [review]

**Pull request merged**